torch_xla/torch_xla.py

```diff
@@ -80,3 +85,27 @@ def manual_seed(seed, device=None):
       If missing the default device seed will be set.
   """
   xm.set_rng_state(seed, device)
+
+
+def launch(
+    fn: Callable,
+    args: Tuple = (),
```

Comment:
Can you add a TODO to make `args` a ParamSpec? I want to take a pass to improve our type annotations, but it'll involve adding `typing` backports as a dependency. `Tuple` is correct and works for this PR.

https://docs.python.org/3/library/typing.html#typing.ParamSpec

Reply:
Oh, I just pressed the merge without noticing the comment. Will update in another PR.